### PR TITLE
fuse: fix crash if empty (None) xattr is read

### LINKED ISSUE #2534: null value in fuse
```
$ borg mount manual_backup/borg_original/ mounted-backup/ --debug --foreground
using builtin fallback logging configuration
Enter passphrase for key /run/media/rpfannsc/backups/manual_backup/borg_original: 
TAM not found and not required
mount data cache capacity: 8 chunks
Mounting filesystem
Initializing llfuse
Calling fuse_mount
Calling fuse_lowlevel_new
Calling fuse_session_add_chan
Calling fuse_session_loop
handler raised <class 'TypeError'> exception (expected bytes, NoneType found), terminating main loop.
Terminated main loop because request handler raised exception, re-raising..
Calling fuse_session_remove_chan
Calling fuse_session_destroy
Local Exception.
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/borg/archiver.py", line 2089, in main
    exit_code = archiver.run(args)
  File "/usr/lib64/python3.5/site-packages/borg/archiver.py", line 2025, in run
    return func(args)
  File "/usr/lib64/python3.5/site-packages/borg/archiver.py", line 91, in wrapper
    return method(self, args, repository=repository, **kwargs)
  File "/usr/lib64/python3.5/site-packages/borg/archiver.py", line 566, in do_mount
    operations.mount(args.mountpoint, args.options, args.foreground)
  File "/usr/lib64/python3.5/site-packages/borg/fuse.py", line 101, in mount
    signal = fuse_main()
  File "/usr/lib64/python3.5/site-packages/borg/fuse.py", line 27, in fuse_main
    return llfuse.main(workers=1)
  File "src/fuse_api.pxi", line 343, in llfuse.main (src/llfuse.c:35968)
  File "src/handlers.pxi", line 624, in llfuse.fuse_getxattr (src/llfuse.c:22336)
TypeError: expected bytes, NoneType found

Platform: Linux ronny 4.10.15-200.fc25.x86_64 #1 SMP Mon May 8 18:46:06 UTC 2017 x86_64 x86_64
Linux: Fedora 25 Twenty Five
Borg: 1.0.10  Python: CPython 3.5.3
PID: 2051  CWD: /run/media/rpfannsc/backups
sys.argv: ['/usr/bin/borg', 'mount', 'manual_backup/borg_original/', 'mounted-backup/', '--debug', '--foreground']
SSH_ORIGINAL_COMMAND: None
```

### PR DESCRIPTION
**Pull request by enkore**
Fixes #2534

### CODE REVIEW

**Comment by enkore** [comment]
Fix confirmed.

**Comment by ThomasWaldmann** [comment]
#2536 master fix #2534 1.0 fix.

File: src/borg/xattr.py [hunk 1]
```diff
@@ -346,10 +356,33 @@ def func(path, name, value, size):
...
+        *path* can either be a path (str or bytes) or an open file descriptor (int).
+        *follow_symlinks* indicates whether symlinks should be followed
+        and only applies when *path* is not an open file descriptor.
+        """
         return []
 
     def getxattr(path, name, *, follow_symlinks=True):
-        pass
+        """
+        Read xattr and return its value (as bytes) or None if its empty.
```
Comment (ThomasWaldmann) [inline src/borg/xattr.py]: it's

Comment (ThomasWaldmann) [inline src/borg/xattr.py]: (maybe i should do a global review for that :)

Comment (enkore) [inline src/borg/xattr.py]: Actually, no :D

**Pull request merged**